recipes/r-glmmml/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = '1.1.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-glmmml
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/glmmML_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/glmmML/glmmML_{{ version }}.tar.gz
+  sha256: 255fe2640933d83ef7ea5813ba8006038c18195147d1f34f47a759210a674dd4
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - libblas
+    - libcblas
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+
+test:
+  commands:
+    - $R -e "library('glmmML')"           # [not win]
+    - "\"%R%\" -e \"library('glmmML')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=glmmML
+  license: GPL-3.0-or-later
+  summary: Binomial and Poisson regression for clustered data, fixed and random effects with
+    bootstrapping.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+# Encoding: UTF-8
```

Comment:
Any reason to keep this comment?

Reply:
Nope, just missed it! Will remove, thanks.